erpnext/docs/user/manual/en/website/shopping-cart.md

```diff
@@ -6,30 +6,44 @@ the items you intend to buy.
 This software displays the price of the product . It also displays shipping
 and handling charges, along with taxes, if applicable.
 
-To set up a shopping cart, go to the selling module.
+To set up a shopping cart, go to the website module.
 
-> Selling > Shopping Cart Settings
+> Website > Shopping Cart Settings
 
-#### Step 1: Enter Company Details and Default Territory.
+#### Enter Company Details and Checkout Details.
 
-![Shopping Cart]({{docs_base_url}}/assets/old_images/erpnext/shopping-cart-1.png)
+<img class="screenshot" alt="Shopping Cart Settings" src="{{docs_base_url}}/assets/img/website/item-website-specs.png">
 
-#### Step 2: Enter Price List, Tax Master and Shipping Rule.
+To make item available on website go to Item master.
 
-![Shopping Cart]({{docs_base_url}}/assets/old_images/erpnext/shopping-cart-2.png)
+> Stock > Item
 
-#### Shopping Cart Display
+#### Enable Item for website.
 
-On the Website, the shopping cart image will be seen below the Item price.
-Customers can click on the cart and enter the amount of quantity they wish to
-buy. The Item number will be stored on the right hand corner of the page, next
-to the flower sign.
+<img class="screenshot" alt="Item" src="{{docs_base_url}}/assets/img/website/item-in-webiste.png">
 
-![Shopping Cart]({{docs_base_url}}/assets/old_images/erpnext/shopping-cart-display-1.png)
+#### Enter Website Specifications for Item.
 
-Click on the flower sign on the right hand side to see the cart details. Click
-on the cart to get the final amount details.
+<img class="screenshot" alt="Website Specifications" src="{{docs_base_url}}/assets/img/website/item-website-specs.png">
 
-![Shopping Cart]({{docs_base_url}}/assets/old_images/erpnext/shopping-cart-display-amount.png)
+#### Add item to shopping cart
```

Comment:
There should be 2 separate section, one for settings to display an item in website and another for how to use shopping cart by the end user.